analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -284,13 +284,23 @@ public void perform() {
         }
     }
 
-    private void writeGmeMarkerFile(File rootGradle) throws IOException, ManipulationException {
+    private void writeGmeMarkerFile(Configuration configuration, File rootGradle) throws IOException, ManipulationException {
         File rootDir = getProject().getRootDir();
         File gmeGradle = new File(rootDir, GME);
 
         if (!gmeGradle.exists()) {
             Files.copy(getClass().getResourceAsStream('/' + GME), gmeGradle.toPath(), StandardCopyOption.REPLACE_EXISTING);
         }
+        if (!isEmpty(configuration.manipulationVersion())) {
+            String gmeGradleString = FileUtils.readFileToString(gmeGradle, Charset.defaultCharset());
+            String currentVersion = gmeGradleString.replaceFirst(
+                    "(?s).*(classpath \"org.jboss.gm:manipulation:)([0-9]+\\.[0-9]+(-SNAPSHOT)??)\".*", "$2");
+            logger.info("Replacing version {} with {} for the ManipulationPlugin", currentVersion,
+                    configuration.manipulationVersion());
+            FileUtils.writeStringToFile(gmeGradle,
+                    gmeGradleString.replaceFirst(currentVersion, configuration.manipulationVersion()),
+                    Charset.defaultCharset());
```

Comment:
I have said before I think it is correct to use `StandardCharsets.UTF_8` as on Windows at least `Charset.defaultCharset()` returns `Cp1252` and I am not well-versed in the differences, at least until this <https://bugs.openjdk.java.net/browse/JDK-8187041> gets accepted.

Reply:
We use `Charset.defaultCharset` everywhere so I'm not convinced its worth changing? We don't even validate on Windows do we?

Reply:
Windows was just an example, although I thought that at least something was running on Windows, PME I guess.. It doesn't just affect Windows, it affects _all_ operating systems, as macOS, is always UTF-8 except in the POSIX C locale and Linux can always be changed via the locale or lang settings.

Probably since the files we write are generally within normal ASCII range it would not fai

Reply:
Given that almost everywhere is using `defaultCharset` at the moment I'm inclined to leave this as is.